### PR TITLE
PXP-6401 JCOIN explorer buttons

### PR DESCRIPTION
**Pull request by paulineribeyre**
Link to Jira ticket if there is one: https://ctds-planx.atlassian.net/browse/PXP-6401

### Environments
JCOIN

### Description of changes
add manifestservice
fix explorer buttons
remove "height" filter because floats are displayed as a list instead of a slider

### CODE REVIEW

**Review by atharvar28 — APPROVED** [review]

File: jcoin.datacommons.io/portal/gitops.json [hunk 1]
```diff
@@ -144,159 +144,192 @@
...
-        "title": "Gender"
+      "filters": {
+        "tabs": [
+          {
+            "title": "Participant",
+            "fields": [
+              "project_id",
+              "gender",
+              "race",
+              "jewish",
```
Comment (gkuffel) [inline jcoin.datacommons.io/portal/gitops.json:176]: Can you remove "Jewish"?

File: jcoin.datacommons.io/portal/gitops.json [hunk 2]
```diff
@@ -144,159 +144,192 @@
...
+              "hispanic",
+              "drug_used",
+              "ever_transferred",
+              "pregnancy_status",
+              "protocol",
+              "weight",
+              "weight_percentage",
+              "bmi",
```
Comment (gkuffel) [inline jcoin.datacommons.io/portal/gitops.json:184]: Remove these also.

**Review by gkuffel — APPROVED** [review]

File: jcoin.datacommons.io/portal/gitops.json (hunk 2 above)
Comment (paulineribeyre) [reply jcoin.datacommons.io/portal/gitops.json:184]: which ones? weight, weight_percentage and bmi?

Comment (gkuffel) [reply jcoin.datacommons.io/portal/gitops.json:184]: Yes.

**Review by gkuffel — APPROVED** [review]

**Pull request merged**